tests/testutils/testutils.go

```diff
@@ -0,0 +1,25 @@
+package testutils
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"sync/atomic"
+
+	"github.com/go-phorce/dolly/algorithms/guid"
+)
+
+var (
+	nextPort    = int32(0)
+	testDirPath = filepath.Join(os.TempDir(), "tests", "trusty", guid.MustCreate())
+)
+
+// CreateURLs returns URL with a random port
+func CreateURLs(scheme, host string) string {
```

Comment:
Needs a unit test. Maybe later. I understand it used rand but at least we can have a unit test that for example checks that the returned string starts with `schema://host` and that the port is within an interval used.

Reply:
Ah on the other hand this is for tests only.

Reply:
this is used in unit tests only. the `tests` folder is excluded from coverage